flow/diff_context.h

```diff
@@ -261,8 +262,11 @@ class DiffContext {
     // determine if subtree has any readback
     size_t position;
 
-    // readback area, in screen coordinates
-    SkIRect rect;
+    // Paint region of the filter performing readback, in screen coordinates.
```

Comment:
Does "screen coordinates" have a solid definition?  Is that in logical pixels or real pixels?

Reply:
It's the physical pixels of target surface. I think the name is used in other places in the engine. It's also referred to as device coordinates sometimes. Same thing, though maybe it should be consistent.